### PR TITLE
Fix/console cleanup

### PR DESCRIPTION
**Pull request by Full-lifey**
# Description

Able to resolve most console warnings and errors, not all of them.

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/Messaging/Messaging.js [hunk 1]
```diff
@@ -123,14 +123,14 @@ function Messaging(props) {
                 let avatar = "";
                 let name = "";
                 let uid = "";
-                conversation.participantAvatars.map(participantAvatar => {
+                conversation.participantAvatars.forEach(participantAvatar => {
```
Comment (Full-lifey) [inline src/components/Messaging/Messaging.js]: This is a better method for just setting variables.

File: src/components/calendar/MeetingDetails.js [hunk 2]
```diff
@@ -1,30 +0,0 @@
-import React from "react";
```
Comment (Full-lifey) [inline src/components/calendar/MeetingDetails.js]: Unused old file created when the calendar was framed out.

File: src/components/mentors/MentorList.js [hunk 3]
```diff
@@ -26,41 +26,29 @@ const MentorList = props => {
...
           };
           holdUsers.push(userDoc);
-          // console.log(userDoc);
         });
-
         setMentors(holdUsers);
       });
   }, []);
 
-  // essentiall think of this as componentWillUnmount
```
Comment (Full-lifey) [inline src/components/mentors/MentorList.js]: Not needed, the first useEffect call automatically unmounts.

**Comment by Full-lifey** [comment]
❗️ ❗️ Just adding this again, please test heavily during review ❗️ ❗️

File: src/components/Messaging/Messaging.js (hunk 1 above)
Comment (LadyKerr) [reply src/components/Messaging/Messaging.js]: I agree

**Pull request merged**